client/my-sites/plan-features/actions.jsx

```diff
@@ -57,6 +63,10 @@ const PlanFeaturesActions = ( {
 				}
 			} );
 		}
+		const isCurrentPlanMonthly = currentSitePlan && isMonthly( currentSitePlan.productSlug );
+		if ( isCurrentPlanMonthly && getPlanClass( planType ) === getPlanClass( currentSitePlan.productSlug ) ) {
```

Comment:
It would be good to come back to this after #14761 lands and use that solution. I say this mostly just to keep a record 🙂